meta-lmp-base/recipes-support/fioconfig/fioconfig_git.bb

```diff
@@ -0,0 +1,42 @@
+DESCRIPTION = "A daemon to handle configuration management for devices in a Foundries Factory"
+HOMEPAGE = "https://github.com/foundriesio/fioconfig"
+SECTION = "devel"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=2a944942e1496af1886903d274dedb13"
+
+GO_IMPORT = "github.com/foundriesio/fioconfig"
+SRC_URI = "git://${GO_IMPORT} \
+	file://fioconfig.service \
+	file://fioconfig-extract.service \
+"
+SRCREV = "665823b724ca653874b7d08fd3029a88ed5e708e"
+
+UPSTREAM_CHECK_COMMITS = "1"
+
+inherit go goarch systemd
+
+do_compile() {
+	cd ${S}/src/${GO_IMPORT}
+	BUILD_COMMIT=`git rev-parse --short HEAD`
+	FIOCONFIG_LDFLAGS="-X ${GO_IMPORT}/internal.Commit=${BUILD_COMMIT}"
+	mkdir -p ${B}/${GO_BUILD_BINDIR}
+	${GO} build -ldflags="${FIOCONFIG_LDFLAGS}" -o ${B}/${GO_BUILD_BINDIR}/fioconfig main.go
+	chmod u+w -R ${B}
+}
+
+SYSTEMD_PACKAGES += "${PN}"
+SYSTEMD_SERVICE_${PN} = "fioconfig.service fioconfig-extract.service"
+
+do_install_append() {
+	install -d ${D}${systemd_system_unitdir}
+	install -m 0644 ${WORKDIR}/fioconfig.service ${D}${systemd_system_unitdir}/
+	install -m 0644 ${WORKDIR}/fioconfig-extract.service ${D}${systemd_system_unitdir}/
+}
+
+# We need aktualizr because we uses its device gateway connectivity and keys
+RDEPENDS_${PN} = "aktualizr"
```

Comment:
I believe we should depend on aktualizr-lite instead (or ${SOTA_CLIENT}, which is how meta-updater decides which OTA client to install, if aktualizr or aktualizr-lite). This is just because on our build we don't actually have aktualizr installed.